stories/props/eventIdAccessor.mdx

```diff
@@ -0,0 +1,10 @@
+import { Canvas, Story } from '@storybook/addon-docs'
+
+# eventIdAccessor
+
+- type: `string | function (event: Object) => string`
+- default: 'id'
+
+The id of the event. Must resolve to a string. Used as the key for the event in the DOM. If not provided, the event will be given a key of 'evt\_<index>'.
```

Comment:
Must it be a string? Many use a numeric as their `id`.

Reply:
good call, thank you